WordPress/Classes/Services/CommentService.h

```diff
@@ -2,6 +2,7 @@
 #import "LocalCoreDataService.h"
 
 extern NSUInteger const WPTopLevelHierarchicalCommentsPerPage;
+extern NSString *commentStatusAll;
```

Comment:
I need to define all the endpoint `status` types, but I'm not sure yet where/how that will be. This will do for now since we're just using `all`.